core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -590,6 +593,13 @@ public List<TopicRequest> selectFilteredTopicRequests(
       }
     }
 
+    if (teamId != null) {
```

Comment:
is teamId and teamSelected not the same ?

Reply:
teamId is a filter option passed in by the coral UI teamSelected is actually the team of the requestor making the call.

So teamId for topic requests can be used to see claims on topics by other teams and removes all requests by your team. 
teamSelected makes sure any non claim topic requests are all from your team